src/server/repository/assessment/assessment/updateAssessment.ts

```diff
@@ -0,0 +1,22 @@
+import { Assessment } from '@meta/assessment'
+
+import { BaseProtocol, DB } from '@server/db'
+import { read } from '@server/repository/assessment/assessment/read'
+
+import { Adapter } from '../../adapter'
+
+export const updateDefaultCycle = async (
+  params: {
+    assessment: Assessment
+  },
+  client: BaseProtocol = DB
+): Promise<Assessment> => {
+  const { assessment } = params
+
+  const assessmentUpdated = await client.one<Assessment>(
+    `update assessment set props = props || '{"defaultCycle": $1~}' where id = $2 returning  *;`,
```

Comment:
we dont want to return the meta_cache here. activity log would be huge

Reply:
so to fix this problem, the update query should not return anything.
At the end of the method, you simply ```return read({id: assessment.id})```, where read method exist in the same folder of this file